src/main/resources/view/PersonListCard.fxml

```diff
@@ -7,30 +7,36 @@
 <?import javafx.scene.layout.GridPane?>
 <?import javafx.scene.layout.HBox?>
 <?import javafx.scene.layout.Region?>
+<?import javafx.scene.layout.RowConstraints?>
 <?import javafx.scene.layout.VBox?>
 
-<HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
+<HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/16" xmlns:fx="http://javafx.com/fxml/1">
```

Comment:
Iirc there could be some issues in the future if the javafx version is changed to 16 in the fxmls because it is higher than the version of javafx being used. (build.gradle says the project is suing version 11)